app/src/components/layout/Header.vue

```diff
@@ -112,6 +120,11 @@
       currentSearchValue: ''
     };
   },
+  watch: {
+        currentSearchValue(value): void {
+            this.$emit('update:currentSearchValue', value);
```

Comment:
Why does the header emit this event?
It seems not to be used anywhere

Reply:
Seems to be old code from testing variants. Gonna remove it